src/Marketplace/MarketplaceScreen.js

```diff
@@ -45,24 +53,20 @@ const styles = StyleSheet.create({
     alignItems: 'flex-end',
     alignSelf: 'flex-end',
   },
-  produceCardsContainer: {
-    display: 'flex',
-    flexDirection: 'column',
-    flexWrap: 'wrap',
-    alignContent: 'center',
-    justifyContent: 'space-around',
-  },
-  produceCard: {
-    padding: 15,
-  },
 });
 
 export default function MarketplaceScreen({ navigation }) {
+  const [allProduce, setAllProduce] = useState([]);
+  const [unsortedProduce, setUnsortedProduce] = useState([]);
   const [produceList, setProduceList] = useState([]);
 
-  const getProduce = () => {
+  const [subscriptions, setSubscriptions] = useState(false);
```

Comment:
Can you remove all of the subscription stuff? Some of it it causing errors (like the subscription button) but generally it just isn't needed anymore.